.gitignore

```diff
@@ -21,3 +21,4 @@ node_modules/
 .DS_Store
 .vscode/
 temp/
+.history/
```

Comment:
i do not see why this would be needed

Reply:
I'm using localhistory plugin for vscode (https://marketplace.visualstudio.com/items?itemName=xyz.local-history)
So that's just to ignore files generated by it.